BSP/STM32F413/Src/BSP_SPI.C

```diff
@@ -0,0 +1,162 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
```

Comment:
Make the name of this file `BSP_SPI.c` (lowercase c)

Reply:
As a side note, `BSP_CAN.c` also needs to be corrected.

Reply:
Still should change this

Reply:
For some reason I can't change the filename. It might be a problem with my WSL configuration.

Reply:
Yeah it's a weird issue. The way I got around it (probably not the proper way) was to delete the file, make a commit, then create a new file with the lowercase 'c' and put all the code in that and commit it again. It's definitely not the best practice but it worked